lib/heroku/cli.rb

```diff
@@ -1,3 +1,8 @@
+if RUBY_VERSION < '1.9.0'
+  $stderr.puts "WARNING: Heroku Toolbelt will require Ruby 1.9+ beginning next week"
+  $stderr.puts "https://github.com/heroku/heroku/pull/1479"
```

Comment:
:+1: @craigkerstiens mentioned the same thing. I already pushed this out and sent an email to all the users I found that used the CLI in the last week under 1.8.7. I'm going to see how it plays out a bit. (Leaving the date out was semi-intentional to give myself some room, but in hindsight I should've just set a solid date)